biothings_annotator/application/configuration/default.json

```diff
@@ -34,7 +34,7 @@
             "host": "0.0.0.0",
             "motd": true,
             "noisy_exceptions": true,
-            "port": 9000,
+            "port": 9001,
```

Comment:
@everaldorodrigo @ctrl-schaff I think we should keep `9000` port as the default here, but set to `9001` port explicitly from the command line in the docker setup (e.g. at the supervisord conf file). 

This way, it's consistent to have `9000` port for either running directly or from docker.